src/App.js

```diff
@@ -4,18 +4,23 @@ import axios from 'axios';
 import HeaderComp from './Frame/HeaderComp/HeaderComp';
 import FreeWeekRotationComp from './Components/FreeWeekRotationComp/FreeWeekRotationComp';
 import SearchFieldComp from './Components/SearchFieldComp/SearchFieldComp';
+import MatchHistoryComp from './Components/MatchHistoryComp/MatchHistoryComp';
+import {getFullUrl} from './Modules/Helper';
 import "./App.css";
 
 class App extends React.Component {
 	state = {
 		champions: null,
-		username: ""
+		username: "",
+		search_error: "",
+		display_history: false,
+		account_id: ""
 	}
 
 	componentDidMount() {
 		// Get all champions from the server when app starts
 
-		let champions_url = "http://ddragon.leagueoflegends.com/cdn/6.24.1/data/en_US/champion.json"; // all champions json
+		let champions_url = "http://ddragon.leagueoflegends.com/cdn/9.14.1/data/en_US/champion.json"; // all champions json
 		axios.get(champions_url).then(res => {
 			console.log("INSIDE CHAMPION LIST AJAX");
```

Comment:
What is it? It doesn't do anything

Reply:
get champion list because the free week fetch and match history fetch only contains champion id's and the entire champion data exists in this file

Reply:
But what does it do with the data?